cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -2089,7 +2089,7 @@
               },
               "content": {
                 "type": "string",
-                "default": "",
+                "default": "''",
```

Comment:
This is fine, but for future reference, I don't think it really matters here. Anything with out a default will show up blank, but "" as a default will show up blank too, so a user can either fill it in or just delete it.